spec/support/omniauth_helpers.rb

```diff
@@ -0,0 +1,17 @@
+def set_omniauth(opts = {})
+  OmniAuth.config.test_mode = true
+
+  OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new({
+    provider: 'google_oauth2',
+    uid: '123545',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.